recipes-ni/ni-rtfeatures/files/ni-rtfeatures.initd

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          ni-rtfeatures
+# Required-Start:    $syslog
+# Required-Stop:
+# Default-Start:     1 3 4 5
+# Default-Stop:
+# Short-Description: nirtfeatures driver utilities
+# Description:       Perform utility functions related to the nirtfeatures
+#                    linux kernel driver.
+### END INIT INFO
+
+DEV_NIRTFEATURES="/dev/nirtfeatures"
+
+
+# Log messages to the console and to the system message buffer.
+log () {
+	local level=$1
+	local msg=${@:2}
+	local logger_stderr=""
+	[ "$VERBOSE" ] && logger_stderr="--stderr"
```

Comment:
@SparkingSpork 
Is it desirable to hide the console output if the user doesn't have `VERBOSE=yes` during boot? Or should we show it all the time?

Reply:
I think showing this all the time is reasonable. It's a single line and is useful information regardless of if there's detailed debugging going on or not.

Reply:
Removed the unused `logger_stderr` logic in latest commits.